```
SystemError(prefix::AbstractString, [errno::Int32])
```

システムコールがエラーコード（`errno` グローバル変数内）で失敗しました。
